omero/util_scripts/Move_Annotations.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2017 University of Dundee & Open Microscopy Environment.
+# All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Moves Annotations from Images to their parent Wells."""
+
+import omero.scripts as scripts
+from omero.gateway import BlitzGateway
+from omero.model import ExperimenterI, \
+                        ImageAnnotationLinkI, \
+                        WellAnnotationLinkI, \
+                        WellI
+from omero.sys import ParametersI, Filter
+from omero.rtypes import rstring, rlong, robject
+from omero.constants.metadata import NSINSIGHTRATING
+
+
+ANN_TYPES = {
+    'Tag': 'TagAnnotationI',
+    'File': 'FileAnnotationI',
+    'Comment': 'CommentAnnotationI',
+    'Rating': 'LongAnnotationI',
+    'Key-Value': 'MapAnnotationI'
```

Comment:
Yes. My assumption is that users who have added annotations via their own scripts / clients will either have given them a useful namespace that can be used to select them in the Move_Annotations script OR they can also write their own script (or update this one) to customise the task.
Although I could add extra types here, I think this adds extra complexity for users and more testing for us which is probably not justified at this stage.